src/components/switch/switch.style.ts

```diff
@@ -9,9 +9,15 @@ import { StyledCheckableInput } from "../../__internal__/checkable-input/checkab
 import StyledSwitchSlider from "./__internal__/switch-slider.style";
 import StyledValidationIcon from "../../__internal__/validations/validation-icon.style";
 import { FieldLineStyle } from "../../__internal__/form-field/form-field.style";
+import { SwitchProps } from "./switch.component";
```

Comment:
same comment here as in `switch-slider` regarding importing props from the style file to the component file rather than the other way round